pkg/cmd/operator/operator.go

```diff
@@ -206,6 +206,10 @@ func (o *OperatorOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.Co
 		cancel()
 	}()
 
+	return o.Execute(ctx, streams, cmd)
```

Comment:
we could just move the above to RunE instead?

Reply:
We had similar discussion [here](https://github.com/scylladb/scylla-operator/pull/1926/files#diff-a1f618d37d8c9d3b43efd39ea637e339593f9f1aee3a4a87ee514a0c0f6f3acb) and I think one with Execute method is clearer.